status-im/status-react
# [#16577] Add missing context tags variants & fix API

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #16577

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)

This PR implements the missing variants of the context-tag variant and matches its API to figma.

![image](https://github.com/status-im/status-mobile/assets/90291778/93b37e74-3644-477a-a806-0721c8dac573)


### Review notes
I noticed some other components are still incomplete:
- `preview-list` lacked of 20 size
- `group-avatar` lacked of 28 size
- `user-avatar` lacked of 28 size

So I implemented them in order to have this PR working and pass design review, but these components should be checked in the same way I did it with this one.

### Testing notes
The previous context-tag uses have been replaced by this new one reworked.

#### Platforms


- Android
- iOS

### Steps to test

- Open Status
- Open quo2.preview -> tags -> context-tags
- Check the variants.


status: ready
